test/e2e/framework/util.go

```diff
@@ -3241,7 +3241,7 @@ func DumpDebugInfo(c clientset.Interface, ns string) {
 
 // DsFromManifest reads a .json/yaml file and returns the daemonset in it.
 func DsFromManifest(url string) (*appsv1.DaemonSet, error) {
-	var controller appsv1.DaemonSet
+	var dscontroller appsv1.DaemonSet
```

Comment:
this is a daemonset, not a controller?
```suggestion
	var ds appsv1.DaemonSet
```